### DISCUSSION TITLE
Caddy over Nginx

### DISCUSSION THREAD

**Discussion by abdurrahmanriyad** [dunglas/symfony-docker#221] [General]
I am just a bit curious to know what's the benefit of using Caddy over Nginx.

**Comment by maxhelias** [comment]
You will find different information on the internet but you have the main points:

- Caddy is very straightforward and simpler configs
- Caddy has automatic HTTPS with Let's Encrypt
- Quick Restart and Rayload Web Server
- Can is easily expandable (like Mercure or Vulcain plugin)

**Reply by abdurrahmanriyad** [reply]
Thanks @maxhelias
